app/scripts/modules/google/src/loadBalancer/configure/internal/gceCreateInternalLoadBalancer.controller.ts

```diff
@@ -218,29 +228,39 @@ class InternalLoadBalancerCtrl extends CommonGceLoadBalancerCtrl implements ng.I
     this.loadBalancer.loadBalancerName = this.getName(this.loadBalancer, this.application);
   }
 
+  public addListener (): void {
+    this.updateSubnetOptions();
+    this.loadBalancer.listeners = this.loadBalancer.listeners || [];
+    this.loadBalancer.listeners.push({subnet: this.subnetOptions[0], ports: 'port1,port2,etc'} as IGceListener);
```

Comment:
Not sure if the port hint is necessary (since we have a help field), but if you do want to include that, it should go as a `placeholder` on the `input` field.